packages/shared/src/hooks/useOnboardingModal.ts

```diff
@@ -51,10 +52,11 @@ export const useOnboardingModal = ({
   };
 
   const onCloseOnboardingModal = () => {
-    if (onboardingMode === OnboardingMode.Auto) {
-      trackEvent({
-        event_name: 'my feed onboarding skip',
-      });
+    if (
+      onboardingMode === OnboardingMode.Auto &&
+      onboardingVersion === OnboardingVersion.V1
+    ) {
+      trackEvent({ event_name: AnalyticsEvent.OnboardingSkip });
```

Comment:
Why do we track the onboarding skip event here as well? Isn't it better to track in a centralized place?

Reply:
The version one doesn't have the same condition to show a confirmation modal when closing/skipping the onboarding. So every "kind" of closing was intentional.